pyperf/_process_time.py

```diff
@@ -26,9 +26,13 @@
     resource = None
 
 
-def get_max_rss():
+def get_max_rss(children=False):
```

Comment:
You may want to make the parameter mandatory:

```suggestion
def get_max_rss(*, children):
```